index.html

```diff
@@ -211,9 +213,24 @@ <h2 class="skill">Skills</h2>
     
 
   </ul>
+</section>
+<section class="contact-form">
+<h2 class="contact-me" id="contact-form">
+  Contact me
+</h2>
+<p id="contact-form">
+  If you have an application you <br>are interested in developing, a <br>feature that you need built or a <br>project that needs coding.<br>
+I’d love to help with it!
+</p>
+<form class="form"action="https://formspree.io/f/{form_id}" method="post" id="contact-form">  
+  <input class="name-email" name="name" id="name" type="text" maxlength="30" placeholder="Abiy Berhe" required >   
+  <input class="name-email" name="Email" id="email" type="email" placeholder="abiyberhe@gmail.com" required>
+  <input class="message" name="message" id="message" type="text" maxlength="500" placeholder="Write your message here..." required> 
```

Comment:
- Great job implementing your form using Formspree 👍🏽 
- I'd like to kindly suggest that we replaced line 228 with `<textarea>` instead of an input field. This is because message input should be <textarea>.

Can we change this? 🤗